mini-bank/user-profile.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>User Profile</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link rel="stylesheet" href="../helpers/css/components.css" />
+    <link rel="stylesheet" href="mini-bank.css" />
+  </head>
+  <body>
+    <div class="container-fixed">
+      <section class="header-component">
+        <div class="navigations">
+          <div class="nav-buttons">
+            <ul>
+                <a href="transactions.html"
+                ><li>Transactions</li></a
+              >
+              <a href="deposits.html"><li>Deposit</li></a>
+              <a href="withdraw.html"><li>Withdraw</li></a>
+            </ul>
+            </ul>
+          </div>
+
+          <a href="user-profile.html">
+            <div class="user-nav">
+              <div class="user-icon"><i class="fas fa-circle-user"></i></div>
+              <div id="user-name" class="user-name">Username</div>
+            </div>
+          </a>
+        </div>
+
+        <div class="balance-wrapper">
+          Bal: &nbsp
+          <div class="user-balance" id="user-balance">0.00</div>
+        </div>
+      </section>
+
+      <div class="page-desc-holder">
+        <h1 class="page-desc">My Profile</h1>
+      </div>
+
+      <div class="header-component">
+        <form id="profile-form">
+          <div class="form-group">
+            <label>Account Name</label>
+            <input type="text" class="form-control" id="user-account-name" disabled/>
+          </div>
+          <div class="form-group">
+            <label>Account Number</label>
+            <input type="number" class="form-control" id="user-account-number" disabled/>
```

Comment:
Only this field should be disabled, because the system will auto-generate a 10-digit account number for the user the first time while creating the account.